drivers/hydrophones/scripts/hydrophones

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
```

Comment:
Is this all legacy code? If so in the future you may want to change the author on the commit to give credit to the original authors